apps/condo/domains/common/hooks/useCurrentBuild.tsx

```diff
@@ -0,0 +1,21 @@
+import useSWR from 'swr'
+import get from 'lodash/get'
+import getConfig from 'next/config'
+
+const {
+    publicRuntimeConfig: { serverUrl },
+} = getConfig()
+
+const REQUEST_KEY = '/api/version'
+const API_ROUTE = `${serverUrl}${REQUEST_KEY}`
+
+async function fetchBuildInfo () {
+    const response = await fetch(API_ROUTE)
```

Comment:
According to documentation:
```suggestion
async function fetchBuildInfo (url) {
    const response = await fetch(url)
```
Yes, it looks strange: a key becomes an url.